java/open-mobile-scene-package/README.md

```diff
@@ -2,19 +2,26 @@
 
 Opens and displays a scene from a Mobile Scene Package (.mspk).
 
-![Open Mobile Scene Package App](open-mobile-scene-package.png)
+![Image of open mobile scene package](open-mobile-scene-package.png)
+
+## Use case
+
+An .mspk file is an archive containing the data (specifically, basemaps and features), used to display an offline 3D scene.
 
 ## How to use the sample
 
-Run the app and allow read permissions.
+When the sample opens, it will automatically display the Scene in the Mobile Map Package.
+
+Since this sample works with a local .mspk, you will need to download the file to your device.
 
 ## How it works
 
-1. Create a `MobileScenePackage` using a path to a local .mspk file.
-1. Use `isDirectReadSupportedAsync()` to check whether the mobile scene package can be read in the archived form (.mspk) or whether it needs to be unpacked.
-1. If the mobile scene package requires unpacking, call `unpackAsync()` and wait for unpacking to complete.
-1. Call `loadAsync()` on the mobile scene package and check for any errors.
-1. When the mobile scene package is loaded, get the first `Scene`and set it to the `SceneView`.
+This sample takes a Mobile Scene Package that was created in ArcGIS Pro, and displays a `Scene` from within the package in a `SceneView`.
+
+1. Create a `MobileScenePackage` using the path to the local .mspk file.
+2. Call `MobileScenePackage.load` and check for any errors.
```

Comment:
This should be loadAsync(). Also, it's a bit confusing because the sample calls loadAsync after creating the done loading listener, but I don't know if it's worth rearranging the instructions for that.